test/service/test_drs.py

```diff
@@ -1,16 +1,28 @@
 import unittest
+from unittest import mock
 
 import requests
+import responses
 
-from azul import drs
+from chalice.config import Config as ChaliceConfig
+from azul import drs, config
 from service import WebServiceTestCase
 
 
 class DataRepositoryServiceEndpointTest(WebServiceTestCase):
 
+    def chalice_config(self):
+        return ChaliceConfig.create(lambda_timeout=15)
+
+    @responses.activate
```

Comment:
While this works here because you're not using moto or other libraries that mock `requests`, I would still want to use `ResponsesHelper` for consistency's sake, to reinforce the pattern and to make future changes to this test easier. It's the same amount of code, just a CM instead of a decorator.